apps/crn-auth-frontend/index.html

```diff
@@ -48,5 +39,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script type="module" src="/src/index.tsx"></script>
```

Comment:
```suggestion
    <script type="module" src="/src/index.tsx" />
```
Seems odd that its a tsx file and that this was added without removing another

Reply:
So, yeah, with regards to pointing the script tag to a tsx file - that's just how Vite works - check out the output files (now in a fixed artifact) to see what it gets converted to. More on this [here](https://vitejs.dev/guide/#index-html-and-project-root). The way Vite works in dev is that they serve transpiled code from memory while replacing the path to a standard .js file for production builds. 

As for not replacing a different script - react scripts append the script tag to the head tag afak

Reply:
Shouldn't this be in the head with the "defer" attribute for best performance (https://gomakethings.com/when-should-you-add-the-defer-attribute-to-the-script-element/#so-when-should-you-use-defer)